string FileName
uint8 IsBreakable
---
uint8 Data